homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -435,3 +410,10 @@ async def async_update(self):
                 _LOGGER.error(
                     "Got exception while fetching the gateway illuminance state: %s", ex
                 )
+
+
+def _get_sensor_description(sensor: str) -> XiaomiMiioSensorDescription:
```

Comment:
If we keep `SENSOR_TYPES` as a dict we don't need this function.

Reply:
Yes, good point.